rust/main/utils/run-locally/src/cosmosnative/cli.rs

```diff
@@ -151,6 +151,9 @@ impl SimApp {
         // TODO: test against the tx result to see if everything was created correctly
         self.tx(vec!["hyperlane", "hooks", "igp", "create", DENOM]);
 
+        const IGP_ADDRESS: &str =
```

Comment:
nit: consts are usually defined at the top of the file (like `GENESIS_FUND`), so they're easy to export elsewhere if needed